axelrod/tests/strategies/test_resurrection.py

```diff
@@ -33,3 +33,30 @@ def test_strategy(self):
         
         # Check for four defections and a cooperation by opponent
         self.responses_test([D], [C] * 5, [D, D, D, D, D, C])
+        
+class TestDoubleResurrection(TestPlayer):
+
+    name = "DoubleResurrection"
+    player = axelrod.DoubleResurrection
+    expected_classifier = {
+        'memory_depth': 1,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        # Starts by Cooperating
+        self.first_play_test(C)
+        
+        actions = [(C, C), (C, D), (D, C), (C, D), (D, C)]
```

Comment:
Can we make sure we have tests that check both 5 round depth behaviors? I think the tests will need to be at least 6 rounds of history to make sure they are triggered.